packages/build/src/plugins/node_version.js

```diff
@@ -25,7 +26,7 @@ const addPluginsNodeVersion = function ({ pluginsOptions, mode, nodePath, userNo
 // the Node.js versions our build system supports and the Node.js versions @netlify/build supports -
 // https://github.com/netlify/pod-workflow/issues/219
 const checkForOldNodeVersions = function ({ pluginsOptions, userNodeVersion, logs, mode }) {
-  if (mode !== 'buildbot' || satisfies(userNodeVersion, '>=12')) return
+  if (mode !== 'buildbot' || satisfies(userNodeVersion, '>=10')) return
```

Comment:
Should we use a specific version, i.e. the minimum minor+patch version we are expecting to support?
In CLI, we've been [using `10.18.0`](https://github.com/netlify/cli/blob/84d79169688f7e57d27db22fd6d569c3e54d731c/package.json#L14). [Same with ZISI](https://github.com/netlify/zip-it-and-ship-it/blob/a36adf0bfcb253ec0271e974752f52a29796476e/package.json#L98).

Reply:
Added it as well as a `TODO` so that we later rely in the version coming from the `package.json`.engines entry. Also, I figure once we hit 100% we can revise this code - https://github.com/netlify/build/blob/main/packages/build/src/plugins/node_version.js#L88-L92 - since we'll always default to running plugins with the system node version if their node version is too old 🤔